packages/attribute-registry-process/src/model/domain/toEvent.ts

```diff
@@ -1,25 +1,9 @@
 import { CreateEvent } from "pagopa-interop-commons";
 import {
   AttributeEvent,
-  AttributeKind,
-  AttributeKindV1,
   Attribute,
-  AttributeV1,
+  toAttributeV1,
 } from "pagopa-interop-models";
-import { match } from "ts-pattern";
-
-export const toAttributeV1 = (attribute: Attribute): AttributeV1 => ({
-  ...attribute,
-  kind: toAttributeKindV1(attribute.kind),
-  creationTime: attribute.creationTime.toISOString(),
-});
-
-export const toAttributeKindV1 = (input: AttributeKind): AttributeKindV1 =>
-  match(input)
-    .with("Declared", () => AttributeKindV1.DECLARED)
-    .with("Verified", () => AttributeKindV1.VERIFIED)
-    .with("Certified", () => AttributeKindV1.CERTIFIED)
-    .exhaustive();
```

Comment:
Moved to the `models` package so that they can be used also in `attribute-registry-readmodel-writer`.